yaml-templates/generic-analysis-properties-file-sonarcloud.yml

```diff
@@ -0,0 +1,42 @@
+# Generic Analysis with SonarCloud - using an existing properties file
+# * If necessary, create an endpoint to connect to SonarCloud
```

Comment:
It would be useful to have a one-liner describing when the template should be used e.g.
`Specific templates exist for MSBuild, Maven and Gradle. Use this template for other types of build.`

Reply:
Looks like the description has been added to the other generic template but not this one, or am I being stupid?

Reply:
Ah,na, you're not, it's a mistake ! Corrected.